types/howler/index.d.ts

```diff
@@ -106,12 +91,7 @@ interface Howl {
 }
 
 interface HowlStatic {
-    new (properties: IHowlProperties): Howl;
+    new (properties: HowlProperties): Howl;
 }
 
 declare let Howl: HowlStatic;
-
-declare module "howler" {
```

Comment:
this changes the semantics of the module. the package it self seems to be exporting these two properties, see https://github.com/goldfire/howler.js/blob/master/dist/howler.js#L2177-L2180